test/xunit.analyzers.tests/AssertEqualStringValueShouldNotUseGenericTypeTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Xunit.Analyzers
+{
+    public class AssertEqualStringValueShouldNotUseGenericTypeTests
+    {
+        public class Analyzer
+        {
+            readonly DiagnosticAnalyzer analyzer = new AssertEqualStringValueShouldNotUseGenericType();
+
+            public static TheoryData<string, string> Foo { get; } = new TheoryData<string, string>
```

Comment:
Yeah, I know there are a few "placeholder" variable names in the source code. I should fix them.